tokio/src/task/coop/mod.rs

```diff
@@ -107,8 +167,21 @@ fn with_budget<R>(budget: Budget, f: impl FnOnce() -> R) -> R {
     f()
 }
 
+/// Returns `true` if there is still budget left on the task.
+///
+/// This function is similar to [`consume_budget()`], but doesn't decrement the
+/// budget when non-zero.
```

Comment:
The wording I suggested was to describe the example, not the function as a whole.

Reply:
I reworked the docs to demonstrate the use case using a better example.